commons/src/main/java/org/dhis2/commons/filters/data/FilterRepository.kt

```diff
@@ -1,24 +1,24 @@
-package org.dhis2.data.filter
+package org.dhis2.commons.filters.data
 
```

Comment:
When merging to develop there is another issue [ANDROAPP-3129](#2250) that adds to this class, We need to take that into consideration.